site/app/controllers/forum/ForumController.php

```diff
@@ -129,6 +129,15 @@ private function isCategoryDeletionGood($category_id){
         }
         return false;
     }
+    /*
+     * Checks if there are a significant amount of announcements and pinned threads and adds a notice
+     * @param $type is adding a pinned thread or making a thread an announcement
+     */
+    private function checkNotifyToUnPin($type) {
+        if ($type && $this->core->getQueries()->getNumberOfPinnedThreadsAndAttachments() > 5 && $this->core->getUser()->accessFullGrading()) {
+            $this->core->addNoticeMessage("Pinned threads drowning new threads? Consider removing some announcements or unpinning some threads.");
```

Comment:
Oh I didn't realize the pin was only on the user side. Ill change it so that only attachments trigger the notice.

Reply:
I never noticed this, but now that I have, I dislike that this terminology runs counter to most other services where the star adds something to your favorites/starred items and the pin is seen globally. Examples of this would be:
* Slack
* Piazza
* vBulletin message boards

Reply:
We did have a long discussion about this over a year ago, and it was a conscious decision (if not unanimous).  I'm going to request that we table this discussion as low priority.  I don't believe any users are reporting any confusion, but I don't want to swap icons on people randomly mid-semester.  We have far more pressing forum VPAT issues to address first.